src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -842,14 +842,6 @@ public void Run()
                 // The workspace has been built for the first time
                 silenceNodeModifications = false;
 
-                // An event handler we can listen to on the NodeViewModel, to update the 
-                // node's informational state. This is required because Errors and Warnings 
-                // currently fire differently from one another. 
-                foreach (NodeModel nodeModel in task.ModifiedNodes)
-                {
-                    nodeModel.OnNodeMessagesClearing();
-                }
```

Comment:
This  was recently added in #12080 but doesn't seem necessary as `OnNodeMessagesClearing` is being called in `ClearErrorsAndWarnings` which keeps the bubble messages in sync with the info states on the node anyway.